apps/game/src/info/intersection.rs

```diff
@@ -193,28 +193,29 @@ fn current_demand_body(ctx: &mut EventCtx, app: &App, id: IntersectionID) -> Wid
     }
 
     let mut batch = GeomBatch::new();
-    let polygon = app.primary.map.get_i(id).polygon.clone();
+    let mut polygon = Tessellation::from(app.primary.map.get_i(id).polygon.clone());
```

Comment:
Here's an example where we just need to draw, so it's easier to work on a `Tessellation`. If we `scale` something down into oblivion, nobody cares